persistent/Database/Persist/Types/Base.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE MultiWayIf #-}
```

Comment:
I would honestly prefer to avoid this extension, since it is completely redundant with `case` expressions.

```suggestion
```